src/Form/AkamaiCacheControlForm.php

```diff
@@ -72,6 +70,14 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
+    $akamai = \Drupal::service('akamai.akamaiservice');
+    global $base_url;
+
+    foreach (explode(PHP_EOL, $form_state->getValue('paths')) as $path) {
+      $akamai->clearUrl($path);
```

Comment:
We should for sure, or at least have some level of awareness of it as it passes through (so we can log, throw an Exception, or whatever). I'm gonna try to leave this kinda stuff to Purge whereever I can though.